src/manage_sql_secinfo.c

```diff
@@ -2854,8 +2886,15 @@ update_scap_cpes ()
   int ret;
 
   full_path = g_build_filename (GVM_SCAP_DATA_DIR,
-                                "nvd-cpes.json",
+                                "nvd-cpes.json.gz",
                                 NULL);
+  if (g_stat (full_path, &state))
+    {
+      g_free (full_path);
+      full_path = g_build_filename (GVM_SCAP_DATA_DIR,
+                                    "nvd-cpes.json",
```

Comment:
`update_scap_cpes_from_json_file` expects a gz file so this will always fail, right? Perhaps just return -1 here.

Reply:
I tried it explicitly once again. The function "gvm_gzip_open_file_reader_fd (..)" in "update_scap_cpes_from_json_file (..)" and the function "fopencookie (..)" within that function can also handle uncompressed files. (Timo knew that.) So it also works for nvd-cpes.json.

Reply:
Ah OK, gzread handles it.